eslint.config.mjs

```diff
@@ -0,0 +1,75 @@
+import globals from 'globals';
+import babelParser from '@babel/eslint-parser';
+import jest from 'eslint-plugin-jest';
+import jestDom from 'eslint-plugin-jest-dom';
+import testingLibrary from 'eslint-plugin-testing-library';
+import { FlatCompat } from '@eslint/eslintrc';
+
+// Useful function tool to more easily merge external shared configuration for a
+// specific files filter.
+function configsForFiles({ files, configs }) {
+    return configs.map(config => ({ ...config, files }));
+}
+
+// Useful to import legacy shared configuration
+const compat = new FlatCompat();
```

Comment:
Oh did they have `FlatCompat` from the start? I don't remember seeing it while I was looking at it a while ago. It could make the migration a bit less painful at least.

Reply:
I'm not so sure :-)

I found it initially by running the migration script (that wasn't doing a super good job).
Also it's mentioned in the migration docs (https://eslint.org/docs/latest/use/configure/migration-guide#using-eslintrc-configs-in-flat-config) but not in a very visible way IMO.